FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -71,6 +64,14 @@ protocol SessionsProvider {
         installations: installations
       )
     )
+    let coordinator = SessionCoordinator(
+      identifiers: identifiers,
+      installations: installations,
+      fireLogger: fireLogger,
+      sampler: SessionSampler(sessionSamplingRate: settings.samplingRate),
+      loggingEnabled: settings.sessionsEnabled
+    )
+    let initiator = SessionInitiator(sessionTimeout: settings.sessionTimeout)
```

Comment:
WIth `timeout` being passed in the initialization phase, we might be stuck at situations where the remote config changes would be applied only in subsequent cold start. I would recommend the configuration change being applied more instantaneously rather than delay. Can we pass the settings service here to the initiator and the initiator can apply the config value at the time of deciding a new session start?

Reply:
That's what I'm unsure about. I believe our design stated updating configs at cold start, so I did it this way. Passing settings is totally possible too, but it needs to be that we intentionally want to possibly change configs every session.